smac/facade/hydra_facade.py

```diff
@@ -264,10 +215,11 @@ def optimize(self) -> typing.List[Configuration]:
             self.tae = ExecuteTARunHydra(ta=self.scenario.ta, run_obj=self.scenario.run_obj,
                                          cost_oracle=self.cost_per_inst, tae=self._tae)
 
-            self.scenario.output_dir = os.path.join(self.top_dir, "smac3-output_%s" % (
+            self.scenario.output_dir = os.path.join(self.top_dir, "psmac3-output_%s" % (
                 datetime.datetime.fromtimestamp(time.time()).strftime('%Y-%m-%d_%H:%M:%S_%f')))
             self.output_dir = create_output_directory(self.scenario, run_id=self.run_id, logger=self.logger)
-        self.rh.save_json(fn=os.path.join(self.top_dir, 'all_runs_runhistory.json'), save_external=True)
+        read(self.rh, os.path.join(self.top_dir, 'psmac3*', 'run_' + str(MAXINT)), self.scenario.cs, self.logger)
```

Comment:
Very minor remark: As this method is by now used by two scripts could we maybe move it to utils?